comms/arduino/arduino.ino

```diff
@@ -70,7 +70,7 @@ struct TPowerData {
   unsigned short mV;
   unsigned short mA;
   unsigned short mW;
-  unsigned long uJ;
+  unsigned long Wh;
```

Comment:
Let's keep using `uJ` and `unsigned long` to limit changes to the communication structure